### DISCUSSION TITLE
Cannot login to LGTM.com with external account after removal of password logins

### DISCUSSION THREAD

**Discussion by guillemj** [community/community#24021] [New to GitHub]
<p>I’ve got the same problem as described in <a href="https://github.community/t/cant-login-to-lgtm-with-external-account-after-removal-of-password-login/137641" class="inline-onebox">Can't login to LGTM with external account after removal of password login!</a>, and cannot login any longer using any of the available methods where I do have an account. Could my github account be linked as well please? :slight_smile:</p>

**Comment by chrisgavin** [comment, marked as answer]
<p>Hi <a class="mention" href="/guillemj">@guillemj</a>,</p>
<p>No problem! I’ve linked your GitHub account to your existing <a href="http://LGTM.com" rel="noopener nofollow ugc">LGTM.com</a> account so you should be able to use the “Login with <a href="http://GitHub.com" rel="noopener nofollow ugc">GitHub.com</a>” option to log in to your account.</p>
<p>Kind regards,<br>
Chris</p>

**Comment by guillemj** [comment]
<p>Hey <a class="mention" href="/chrisgavin">@chrisgavin</a>, thanks that works now, much appreciated!</p>
